website/src/components/section/Home/data.js

```diff
@@ -30,49 +30,49 @@ export const advancedLitmusData = [
     icon: 'img/AdvanceLitmus/advanced.svg',
     heading: 'Advanced User Guides',
     description: 'Learn about Litmus containing the user guide and architecture.',
-    url: '/404'
+    url: '/docs/next/user-guides/overview'
```

Comment:
Can we have these links pointing to current-docs instead of master docs otherwise, it will always be pointing to master docs.

Reply:
If we change the link to point to `/docs` instead of `/docs/next/` it will only work post release